webpack/assets/javascripts/react_app/redux/reducers/notifications/notifications.test.js

```diff
@@ -49,4 +56,32 @@ describe('notification reducer', () => {
       })
     ).toMatchSnapshot();
   });
+
+  it('Should handle SUCCESS', () => {
+    const { SUCCESS } = actionTypeGenerator(types.NOTIFICATIONS);
+
+    const action = {
+      type: SUCCESS,
+      payload: {
+        notifications,
+      },
+    };
+    expect(reducer(stateWithNotifications, action)).toMatchSnapshot();
+  });
+
+  it('Should handle FAILURE', () => {
```

Comment:
How about using `testReducerSnapshotWithFixtures` from our testsHelper?

Reply:
originally I thought to add it in #7138, though I see no problem adding it here :+1:

Reply:
it seem a lot of work to update the convention of all tests,
which is a little bit out of the scope of this PR, would it be ok to change it in #7138 ?